components/camel-braintree/src/main/docs/braintree-component.adoc

```diff
@@ -74,14 +74,15 @@ with the following path and query parameters:
 [width="100%",cols="2,5,^1,2",options="header"]
 |===
 | Name | Description | Default | Type
-| *environment* (common) | *Required* The environment Either SANDBOX or PRODUCTION |  | String
+| *environment* (common) | The environment Either SANDBOX or PRODUCTION |  | String
```

Comment:
The documentation is generated automatically. From your code I see that environment is still required, so probably you modified this by hand.

https://github.com/nofuss/camel/blob/4523c406859d25aa3a47109cc5d350ec14c88a5b/components/camel-braintree/src/main/java/org/apache/camel/component/braintree/BraintreeConfiguration.java#L50

Reply:
I I think I made some mistakes when preparing the pull request - reviewing now

Reply:
I had left out a couple commits when preparing the pull request, so now the code and generated documentation should align